package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "estree",
+  "version": "1.0.0-pre",
```

Comment:
The field is explicitly semver. And since this effort is committed to 100% backwards compatibility, that means the major version will never change.  We could try using minor for that purpose, but it would break down as soon as we added something in between ECMAScript releases. I think it's better to avoid cleverness here.